### DISCUSSION TITLE
Listview bindingerror children of observablecollection in viewmodel

### DISCUSSION THREAD

**Discussion by baswelsink** [dotnet/maui#7165] [Q&A] [answered]
Hi,

I have a question, I'm playin around with .NET MAUI but I'm running into an issue which I do not understand. I have a XAML page for which I've set the BindingContext to be equal to a viewmodel. When attaching the observablecollection to the itemssource of the listview the items are properly displayed. Still I get a binding error where it seems it's looking for the properties in the viewmodel class and not the children of the observable collection. I'm lost, anybody any idea what I'm doing wrong? 

XAML
![image](https://user-images.githubusercontent.com/20791767/168312891-e467fc0c-6fcc-4347-9cd3-db151d59e799.png)

ViewModel
![image](https://user-images.githubusercontent.com/20791767/168313261-3c34f807-c349-45d6-b0e8-b2181cec79ea.png)

BindingError
![image](https://user-images.githubusercontent.com/20791767/168313335-ad7215ee-f0d5-4df5-9678-b77ac5d1c570.png)

**Comment by BinaryAssault** [comment, marked as answer]
Try moving the data type to the grid
````
<Grid x:DataType="local:MyFile">
````

Also, a suggestion, ditch the ListView and use a CollectionView, doing that alone might fix your issue

**Reply by baswelsink** [reply]
Hi, thanks for the suggestion, moved to data type to the grid but doesn't seem to solve the issue. Will try a CollectionView later, I think the code is fine since the binding does work and in UWP this is not an issue. Maybe some bug in a code analyzer...

**Reply by baswelsink** [reply]
Indeed, when using the CollectionView there are no binding errors, thanks!

**Reply by Yo0oo0oY** [reply]
Could you please tell me how you finally resolved these errors
